neqo-http3/src/server.rs

```diff
@@ -35,6 +37,9 @@ impl ::std::fmt::Display for Http3Server {
 }
 
 impl Http3Server {
+    /// # Errors
+    /// Making a `neqo-tracaction::connection` may produce an error. This can only be a crypto error if
```

Comment:
I'm still not sure that this is correct either.  There is no `neqo-transaction`, and if there was, it would be `neqo_transaction`.

```suggestion
    /// Making a `neqo-transaction::connection` may produce an error. This can only be a crypto error if
```

Reply:
pretty sure this is meant to be "transport"

Reply:
this was suppose to be transport :)